less/admin/AppearancePage.less

```diff
@@ -34,3 +35,200 @@
     line-height: 1;
   }
 }
+.StatisticsWidget-table {
```

Comment:
I don't think this all should be here: this seems like statistics extension stuff